site/docs/master/restic.md

```diff
@@ -160,6 +160,86 @@ You're now ready to use Velero with restic.
 
 ## Back up
```

Comment:
I know you didn't add this, but please make this `Backup`.

Reply:
"Backup" is the noun and "Back up" is the verb. Here we are listing steps to perform back up, using the verb and not the noun, correct?

Reply:
I think you're right. It might be good to make it `To back up` and `To restore` (next section).